projects/plugins/jetpack/modules/seo-tools/class-jetpack-seo-posts.php

```diff
@@ -80,7 +108,18 @@ public static function register_post_meta() {
 			),
 		);
 
-		register_meta( 'post', self::DESCRIPTION_META_KEY, $args );
+		$html_title_args = array(
+			'type'         => 'string',
+			'description'  => __( 'Custom title to be used in HTML <title /> tag.', 'jetpack' ),
```

Comment:
I'm assuming that the output of this will properly escape the `&gt;` and `&lt;` of the `<title />` in here and we don't need to escape it in the string to be translated?

Reply:
AFAIK it does not cause problems during translation, and `esc_html__` is used in PHP HTML code to prevent issues when displaying the code. As these args are passed to React code, which handles strings by itself, it's not needed to escape any chars here.